Lib9c/Model/Item/Costume.cs

```diff
@@ -84,6 +87,23 @@ public override IValue Serialize()
         }
 #pragma warning restore LAA1002
 
+        public override IValue SerializeLegacy()
+        {
+#pragma warning disable LAA1002
+            var innerDictionary = new Dictionary<IKey, IValue>
+            {
+                [(Text) LegacyEquippedKey] = equipped.Serialize(),
+                [(Text) LegacySpineResourcePathKey] = SpineResourcePath.Serialize(),
+                [(Text) LegacyCostumeItemIdKey] = ItemId.Serialize()
+            };
+            if (RequiredBlockIndex > 0)
+            {
+                innerDictionary.Add((Text) RequiredBlockIndexKey, RequiredBlockIndex.Serialize());
```

Comment:
Is it okay to use `RequiredBlockIndexKey` though there is `LegacyRequiredBlockIndexKey`?

Reply:
yes, because Costume already used in `RequiredBlockIndexKey`.